README.md

```diff
@@ -1,2 +1,189 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-ok43.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Walkthrough Demo <a name="live-demo"></a>
+
+- [Walkthrogh Demo Link](https://www.loom.com/share/57937b2642d94bf49067ca61a3fba8c2?sid=f3e33c05-7323-4027-b29f-f66b36c0f867)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To start and run this project
+```
+bundle install
+```
+```
+rails s
+```
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- IDE to edit and run codes (Strongly recommend Visual Studio Code 🔥).
+- Git.
+- PostgreSQL installed
+
+
+### Install & Setup
+
+To setup and install this project, follow the below steps:
+- Clone this project by the command: 
+
+```
+$ git clone https://github.com/Theodoraldo/Budget_app.git
+```
+
+- Then switch to the project folder by the bellow query:
+
+```
+$ cd Budget_app
+```
+
+- Install the required dependecies with `bundle install`
+- Install npm dependencies with `npm install`
+- Run `rails db:setup`
+- Run `rails db:migrate`
+- With Ruby on Rails and PostgreSQL installed in your environment, you can run `rails server` or `rails s` to run the website.
+
+### Tests <a name="run-tests"></a>
+
+- Run `rails db:seed RAILS_ENV=test` to seed the testing database
+- Run `rspec`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Theodore Asimeng-Osei**
+
+- GitHub: [@githubhandle](https://github.com/Theodoraldo)
+- Twitter: [@twitterhandle](https://twitter.com/AsimengOse33947)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/theodoreasimeng/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Use file upload for saving image instaed of url**
+- [ ] **Add more functionality like Javascript to make app interactive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Theodoraldo/Budget_app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project give it a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I would like to acknowledge Gregoire Vella for such an awesome design and documentation.
+- Thanks to Microverse and anyone who contributed to the success of this project. If we am to mention names, this readme cannot hold the list 😁😁.
```

Comment:
- [ Required ] Good job adding this section as requested by the previous reviewer, however, please note you forgot to include the links that would take a user to [Gregoire Vella's](https://www.behance.net/gregoirevella) profile. I would like to suggest that you edit this section as shown below so as to make your acknowledgments section more professional.

## Sample attribution
I would like to acknowledge [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) for such an awesome design and documentation.